drivers/wifi/eswifi/Kconfig.eswifi

```diff
@@ -22,4 +22,11 @@ config WIFI_ESWIFI_THREAD_PRIO
 	  This option sets the priority of the esWiFi threads.
 	  Do not touch it unless you know what you are doing.
 
+config WIFI_ESWIFI_SHELL
+	bool "esWiFi shell"
+	default n
```

Comment:
Value is "n" by default, this line can be omitted

Reply:
thanks, fixed now